industrial_taxonomy/pipeline/glass_house/flow.py

```diff
@@ -0,0 +1,196 @@
+from typing import List
```

Comment:
I've gone with no because a flow file is generally just a flow class so a top-level description would just duplicate any info in the Flow's docstring. Does that seem reasonable?